### ISSUE TITLE
Peekaboo "run_av error"

### ISSUE THREAD

**Issue by MigliS** [scVENUS/PeekabooAV#6]
Hi Felix,

I'm really sorry to open up a new issue. Now that all is working fine, I get Peekaboo error in my maillog.
```(!)Peekaboo-Analysis av-scanner FAILED: run_av error: ask_daemon_internal: Exceeded allowed time at (eval 749) line 657.\n```

Not sure, where to find this configuration, but I think it'll be in the amavis configuration.

**Comment by Jack28** [comment]
Hi Michael,
no worries. Issues help others too who run into problems.

Please check whether amavis has filesystem permissions to access the socket file.

Since you don't use Ubuntu I can't really help. Please check if you've followed/replicated the steps performed by the installer and report back.

**Comment by MigliS** [comment]
Amavis should have permissions to access the *.sock-file.
I'm going to try this at home. After that, maybe I'm able to compare the two installations.

**Comment by MigliS** [comment]
Hi Felix,
Would it help if I could send you my log?

**Comment by Jack28** [comment]
If you could attach the mail log and peekaboo log most of the relevant information should be in there.

For debugging purpose I suggest the following:
Become peekaboo user `sudo su -s /bin/bash amavis` and try to connect to the socket using `socat STDIN UNIX-CONNECT:/var/run/peekaboo/peekaboo.sock`. You should then see the greeting message `Hallo das ist Peekaboo`. At this point you could already give a local path to files that will then be scanned recursively if readable by the peekaboo user.

Make sure `/etc/amavis/conf.d/15-av_scanners` contains the correct entry to use Peekaboo and find the socket.

In addition to that you could submit a file to cuckoo manually. Become the `peekaboo` user and run `cuckoo submit /path/to/file`. This should give you the job id which you can follow at the cuckoo web ui.

Let me know how far you got.

**Comment by MigliS** [comment]
Hi Felix,

Yesterday, I was able to start peekaboo in the right way with the service.
It was absolutely my fault.
I wasn't sure, if the patch of amavisd was going to ruin the whole configuration. But I tried and could restart amavisd successfully.
After that I only had trouble, that amavis didn't upload the attachment.

Also this problem was very easy to solve. I didn't know I need to change the path in the amavisd.conf to the same path, like in the peekaboo.conf.

Only a few minutes later, incoming mails did go trough cuckoo.
So it's working now!!! :D


But there's still stuff i couldn't figure out. For example, where I can add files, to send to cuckoo, or where I can change the peekaboo signatures.
